docs/ui/components/Sidebar/SidebarFooter.tsx

```diff
@@ -47,6 +52,13 @@ export const SidebarFooter = () => {
         Icon={ChangelogIcon}
         isExternal
       />
+      <SidebarSingleEntry
+        secondary
+        href="http://eepurl.com/hk1tCn"
+        title="Newsletter"
+        Icon={Mail01DuotoneIcon}
```

Comment:
```suggestion
        Icon={Mail01Icon}
```

Other icons in the footer are non duotone, and we should stick to those variants.

Reply:
Fixed it